### PR TITLE
Update readme for istio install

### PR DESCRIPTION
**Pull request by richardwxn**
The readme looks a bit outdated also it is confusing regarding the usage between ./setup_istio.sh and ./setup_istio_release.sh, trying to 1. reorganize it. 2. update the deprecated part 3. encourage to use ./setup_istio_release.sh across. 

 can you help take a look? @howardjohn

### CODE REVIEW

File: perf/istio-install/setup_istio.sh [hunk 1]
```diff
@@ -151,8 +141,8 @@ function install_istio() {
           "$WD/setup_prometheus.sh" ${DIRNAME}
       fi
       if [[ -n "${MIXERLESS}" ]]; then
-        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/metadata-exchange_filter.yaml
-        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/proxy/master/extensions/stats/testdata/istio/stats_filter.yaml
+        kubectl -n istio-system apply -f https://raw.githubusercontent.com/istio/istio/master/tests/integration/telemetry/stats/prometheus/testdata/metadata_exchange_filter.yaml
```
Comment (mandarjog) [inline perf/istio-install/setup_istio.sh]: As we know these are version sensitive now. Is this going to work with both master and 1.4 of tools?

File: perf/istio-install/setup_istio.sh [hunk 2]
```diff
@@ -26,23 +26,13 @@ mkdir -p "${WD}/tmp"
 
 release="${1:?"release"}"
 
-if [[ "${release}" == *-latest ]];then
```
Comment (mandarjog) [inline perf/istio-install/setup_istio.sh]: -dev is similarly resolved some place else. Can you check if it done in one of the other files?

File: perf/istio-install/setup_istio.sh (hunk 1 above)
Comment (richardwxn) [reply perf/istio-install/setup_istio.sh]: this is legacy part now, I would remove it

File: perf/istio-install/setup_istio.sh (hunk 2 above)
Comment (richardwxn) [reply perf/istio-install/setup_istio.sh]: yes, it is resolved in other PR already, would update accordingly

**Comment by ericvn** [comment]
@richardwxn Just trying to go back and clean up some older PRs. What should we do with these changes? Rebase and get them in? I think #828 and #1012 overlap on some of the files.

**Pull request closed without merging**